avocado/core/safeloader/imported.py

```diff
@@ -105,8 +105,8 @@ def get_symbol_module_path_from_statement(statement, name_index=0):
         return symbol, module_path
```

Comment:
Can you please explain better on commit message what is the limitation?

Reply:
Sure, I'll add the comment I'm removing to the commit message, that is:

"Currently, ImportedSymbol.from_statement() only supports the first symbol imported in the statement.  That's why where the final_name is used instead for the symbol, and the (correct) module comes from its utility method."